tests/helper/util.ts

```diff
@@ -93,23 +101,19 @@ export function parseRegex(regex: RegExp): LiteralAST {
 	return literal;
 }
 
-export function getLeadingSpaces(string: string) {
+export function getLeadingSpaces (string: string) {
 	return /^\s*/.exec(string)?.[0] ?? '';
 }
-export function getTrailingSpaces(string: string): string {
+export function getTrailingSpaces (string: string): string {
 	return /\s*$/.exec(string)?.[0] ?? '';
 }
 
-export function formatHtml(html: string): string {
-	return Prettier.format(html, {
-		printWidth: 100,
-		tabWidth: 4,
-		useTabs: true,
-		htmlWhitespaceSensitivity: 'ignore',
```

Comment:
I don’t see this rule in `.prettierrc`. I know it wasn’t there in the first place, but if we get rid of all this code, shouldn’t we add it there?

Reply:
Yeah that's a good point, I'll add the option back